### PR TITLE
Wf training

### PR DESCRIPTION
**Pull request by btalberg**

### CODE REVIEW

**Comment by btalberg** [comment]
To run Alphapose, you'd need to set AWS creds in the container. I do this by making a volume with my AWS config available in the container.

*Alphapose Training*
```
docker run -ti --rm --gpus all --ipc=host \
    --name alphapose-training \
    -v ${HOME}/.aws/credentials:/root/.aws/credentials:ro \
    -v /data/alphapose-training/data:/build/AlphaPose/data \
    -v /data/alphapose-training/exp:/build/AlphaPose/exp \
    -v /data/alphapose-training/pretrained_models:/build/AlphaPose/pretrained_models \
    wildflowerschools/wf-deep-docker:cuda10.2-alphapose-base-v8 \
    train \
    --detector wfyolov4 \
    --img-size 256 \
    --dataset-type wfcoco17 \
    --num-classes 1 \
    --epochs 500 \
    --dataset s3://wf-sagemaker-us-east-2/final_training_sets/wf-coco-gt-alphapose-v0.3.tar.gz \
    --batch-size 8 \
    --pretrained fast_421_res152_256x192
```

*Alphapose Inference*
```
export VERSION=8
docker run -ti --rm --gpus all --ipc=host \
    --name alphapose-inference \
    -v ${HOME}/.aws/credentials:/root/.aws/credentials:ro \
    -v /data/alphapose-training/data:/build/AlphaPose/data \
    -v /data/alphapose-training/pretrained_models:/build/AlphaPose/pretrained_models \
    wildflowerschools/wf-deep-docker:cuda10.2-alphapose-base-v8 \
    inference \
    --detector wfyolov4 \
    --pose-model wf_res152_256x192_yolov4 \
    --tracker-model jde_1088x608
```

File: darknet-training.dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+FROM wildflowerschools/wf-deep-docker:cuda10.2-opencv-base-v0
+
+RUN apt install python3-venv -y && \
+    cd /build && git clone https://github.com/WildflowerSchools/darknet
+
+WORKDIR /build/darknet
+
+RUN sed -i 's/GPU=0/GPU=1/g' Makefile && \
```
Comment (optimuspaul) [inline darknet-training.dockerfile]: what is going on here?

Comment (optimuspaul) [reply darknet-training.dockerfile]: If I understand correctly you can do this:

`make GPU=1 CUDNN=1 CUDNN_HALF=1 OPENCV=1`

and get the same results

Comment (btalberg) [reply darknet-training.dockerfile]: Ah, interesting, I hadn't tried that approach. I followed an install guide on the darknet website that walks you through [editing the Makefile](https://pjreddie.com/darknet/install/), ergo the `sed` commands.

**Review by optimuspaul — APPROVED** [review]

**Pull request merged**